### PR TITLE
add Layla tune

### PR DESCRIPTION
**Pull request by metehan-arslan**
Hi

Tried to recreate Layla tune from Eric Clapton, wanted to whole riff but it was long so I only added lick part. Feel free to comment and correct.
NB: I don't have good musical ear, sorry for any errors 😅

Notes:
A6, 40ms
REST, 10ms
C7, 50ms
REST, 20ms
D7, 50ms
REST, 10ms
F7, 50ms
REST, 10ms
D7, 50ms
REST, 20ms
D7, 150ms

### CODE REVIEW

File: movement/movement_custom_signal_tunes.h [hunk 1]
```diff
@@ -84,4 +84,21 @@ int8_t signal_tune[] = {
...
+    BUZZER_NOTE_A6, 4,
+    BUZZER_NOTE_REST, 1,
+    BUZZER_NOTE_C7, 5,  
+    BUZZER_NOTE_REST, 2,
+    BUZZER_NOTE_D7, 5,
+    BUZZER_NOTE_REST, 1,
+    BUZZER_NOTE_F7, 5,
+    BUZZER_NOTE_REST, 1,
+    BUZZER_NOTE_D7, 5,
+    BUZZER_NOTE_REST, 2,
```
Comment (mooskagh) [inline movement/movement_custom_signal_tunes.h:98]: Another C7 is missing between two D7s.

File: movement/movement_custom_signal_tunes.h [hunk 2]
```diff
@@ -84,4 +84,21 @@ int8_t signal_tune[] = {
...
+    BUZZER_NOTE_D7, 15,
```
Comment (mooskagh) [inline movement/movement_custom_signal_tunes.h:99]: In the song, the duration in the first repeat is dotted quarter (i.e. 30), in the second it's plain quarter (i.e. 20), so you may want to make it at least 20.

File: movement/movement_custom_signal_tunes.h [hunk 3]
```diff
@@ -84,4 +84,21 @@ int8_t signal_tune[] = {
 };
 #endif // SIGNAL_TUNE_KIM_POSSIBLE
 
+#ifdef SIGNAL_TUNE_LAYLA
+int8_t signal_tune[] = {
+    BUZZER_NOTE_A6, 4,
+    BUZZER_NOTE_REST, 1,
+    BUZZER_NOTE_C7, 5,  
+    BUZZER_NOTE_REST, 2,
```
Comment (mooskagh) [inline movement/movement_custom_signal_tunes.h:92]: Any reason why some rests have duration of 2, while others have 1 (and also the first note is shorter than the rest)? At least on paper, all note durations are actually same.

File: movement/movement_custom_signal_tunes.h (hunk 1 above)
Comment (metehan-arslan) [reply movement/movement_custom_signal_tunes.h:98]: thanks for pointing this out! I was wondering why last notes part sounded off

File: movement/movement_custom_signal_tunes.h (hunk 2 above)
Comment (metehan-arslan) [reply movement/movement_custom_signal_tunes.h:99]: got that 👍

File: movement/movement_custom_signal_tunes.h (hunk 3 above)
Comment (metehan-arslan) [reply movement/movement_custom_signal_tunes.h:92]: I tried tweaking timings with my ear, last night reducing first note and dividing some notes based on timing sounded closer to my ear -- never trust biological organs 😓

**Comment by metehan-arslan** [comment]
Thanks for the review @mooskagh, it sounds much better now.
here are the before and after:

https://github.com/user-attachments/assets/2b368eae-4a95-4288-a05b-a80d38533995

https://github.com/user-attachments/assets/3c6f281b-792a-4681-b825-967ff3a85888

**Review by matheusmoreira — APPROVED** [review]
Nice!

**Pull request merged**